platform/accessibility/guidance/defect-severity-rubric.md

```diff
@@ -1,102 +1,137 @@
 * Content that is too far removed from an explanation, or with no explanation at all
-* Missing landmark HTML. This includes banner, navigation, main, aside, footer elements, or divs with role attributes.
-* Improper headings, nesting
-* Improper ARIA attributes, missing IDs
-* [Not managing focus for single-page applications](https://adhocteam.us/2018/02/20/developer-driven-focus-management-for-single-page-applications/)
-
-### Potential Users Impacted
-* Keyboard navigation
-* Screen reader or other assistive device
+* Missing landmark HTML. This includes:
+  * `<banner>`
+  * `<navigation>`
+  * `<main>`
+  * `<aside>`
+  * `<footer>`
+  * divs with `role` attributes.
+* Improper heading nesting. Adding an `<h4>` as the first heading underneath an `<h2>` is an example of improper nesting. The [WebAIM WAVE](https://wave.webaim.org/) or [headingsMap](https://chrome.google.com/webstore/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi?hl=en) browser plugins  provide visual representations of heading nesting.
+
+### Potential impact:
+
+* Users who navigate by keyboard
+* Users who navigate by screen reader or other assistive device
 * Users who have partial vision or are blind
 * Users who are hard of hearing or deaf
 * Users with identified short-term memory or other cognitive issues
 
-### Time to Fix
-* Issue should be prioritized for fix 1-2 sprints post-production launch
+## 508-defect-3
+
+**⚠️ Minor. Should be fixed in 1-3 sprints post-launch.**
+These issues have the potential to disrupt the user experience and should be fixed as time allows in the first 1-3 sprints post-launch.
+
+### Types of issues:
 
-## 508-defect-3 (Minor)
+* Errors in early release tools like [axe-coconut](https://www.deque.com/axe/axe-for-web/early-release/)
+* Best practice warnings in automated scanning tools like [aXe browser plugin](https://www.deque.com/axe), [SortSite](https://www.powermapper.com/products/sortsite/), [AATT](https://github.com/paypal/AATT)
+* [Invalid HTML](https://validator.w3.org/) (non-semantic, badly formed)
 
-### Type of issue:
-* Minor errors in automated testers like [aXe plugin](https://www.deque.com/axe), [SortSite](https://www.powermapper.com/products/sortsite/), [AATT](https://github.com/paypal/AATT)
-* Invalid HTML (non-semantic, badly formed)
+### Potential impact:
 
-### Potential Users Impacted
-* Screen reader, assistive devices
-* Low vision users
-* Keyboard users
+* Users who navigate by screen reader or other assistive device
+* Users who have partial or low vision
+* Users who navigate by keyboard
 
-### Time to Fix
-* Issue should be prioritized for next iteration (often 3 sprints in length) or sooner if time allows
-* These issues are often good items for engineers to pick up when they have a few extra hours at the end of a sprint
+## 508-defect-4
 
-## 508-defect-4 (Trivial)
+**✔️ Trivial. Consider fixing or exploring in 2-4 sprints post-launch.**
```

Comment:
Consider all caps for CONSIDER.

Reply:
All caps was feeling loud. I think I'm going to keep it as is.